### DISCUSSION TITLE
Support for Azure ActiveDirectory login for Postgresql

### DISCUSSION THREAD

**Discussion by aelghoneimy** [dbeaver/dbeaver#19394] [Ideas]
As you know, Microsoft supports Postgresql services in Azure. It also integrates these services with the built-in Azure ActiveDirectory authentication.

Like how Azure SqlServer can authenticate with AD Integrated and MFA, it would be very convenient to have the same option for Postgresql.

![image](https://user-images.githubusercontent.com/4632474/224710761-833d9d71-e17d-4d9a-a1db-04d78d1bbf75.png)

**Comment by E1izabeth** [comment]
Thank you for the idea

**Comment by LonwoLonwo** [comment]
Azure ActiveDirectory login for Postgresql supported by DBeaver PRO versions https://dbeaver.com/

You can try it for free - https://dbeaver.com/trial/

![2023-03-13 16_32_34-Connect to a database](https://user-images.githubusercontent.com/45152336/224724193-b2a4577f-ab2c-482f-be70-f1173098891c.png)

**Comment by aelghoneimy** [comment]
@LonwoLonwo which versions of DBeaver pro supports this? I downloaded the lite and it's not there

![image](https://user-images.githubusercontent.com/4632474/224730423-4b326cf3-764c-4d70-ab33-d68db3b494df.png)

**Reply by djpremier** [reply]
Here is comment that "This feature is available in Lite, Enterprise, Ultimate and Team editions only."

https://dbeaver.com/docs/dbeaver/Authentication-Azure-AD/

**Comment by LonwoLonwo** [comment]
For now, it is available only in the Ultimate Edition.

Probably, you can find info in the Internet - how to use PostgreSQL driver properties to connect via Azure

**Reply by ridicolos** [reply]
Are there any info when this will be available in Enterprise or the Community-Version?

**Comment by djpremier** [comment]
Hi, is there any expectation/estimate of having this authentication mode in the Community version?

**Comment by ridicolos** [comment]
Would really love to see this feature across all version of DBeaver.

We are using a combo of DBeaver Community and DBeaver Enterprise and really need a way to authenticate with using the azure acitve directory.
